test/simulation/actions.go

```diff
@@ -400,7 +400,7 @@ func getInfo(ctx context.Context, n *xudtest.HarnessNode) (*xudrpc.GetInfoRespon
 }
 
 func openBtcChannel(ctx context.Context, ln *lntest.NetworkHarness, srcNode, destNode *lntest.HarnessNode) (*lnrpc.ChannelPoint, error) {
-	openChanStream, err := ln.OpenChannel(ctx, srcNode, destNode, 15000000, 0, false)
+	openChanStream, err := ln.OpenChannel(ctx, srcNode, destNode, 15000000, 7500000, false)
```

Comment:
I think in the future we might want to open channels using xud's `openchannel` API. I'm good with not doing it in this PR. Xud's openchannel doesn't currently support push amount.

Reply:
Yup that would be a good idea indeed.